apps/trustlab/src/payload/access/abilities.js

```diff
@@ -0,0 +1,60 @@
+import { checkRole } from "./checkRole";
+import { ROLE_ADMIN, ROLE_AUTHOR, ROLE_EDITOR } from "./roles";
+
+export const canManageContent = (user) => {
+  // Admin and editors can manage any content
+  if (checkRole([ROLE_ADMIN, ROLE_EDITOR], user)) {
+    return true;
+  }
+
+  // Everyone else can only manage their own content
+  return {
+    createdBy: {
+      equals: user.id,
+    },
+  };
+};
+
+export const canManagePages = (user) =>
+  checkRole([ROLE_ADMIN, ROLE_EDITOR], user);
+
+export const canManageSiteSettings = (user) => checkRole([ROLE_ADMIN], user);
+
+// TODO: what happens on delete? cascade or not?
```

Comment:
```suggestion
// TODO(kelvinkipruto): what happens on delete? cascade or not?
```

Add a name so we know who to talk about about this or who is going to do it.